tests/perf/perf_tests_perf.cc

```diff
@@ -0,0 +1,68 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
```

Comment:
> These are benchmarks which just test minimal version of the various
> PERF_TEST flavors, with mostly empty test bodies. It is intended to
> test the overhead of the test framework, so that changes can be made
> without any large regressions. There are a lot of cases because
> internally the main perf test loop has different flavors depending
> on async or not, returning an iteration count or not, etc.

Please enrich the changelog with sample output.

Reply:
Good idea, sample out here (from after the coroutine change):

https://github.com/scylladb/seastar/pull/2588#issuecomment-2552232283

Reply:
Looks like you forgot to add it to the cover letter.

Reply:
It has been added it to the cover letter now.